### DISCUSSION TITLE
[question] k8s documentation

### DISCUSSION THREAD

**Discussion by snwnst** [saleor/saleor#9169] [Deployment]
Is there any official documentation for the deployment on kubernetes?

**Comment by emrebyimrn** [comment]
yeahh looking for the same ,
can anyone help us!!!

**Comment by rodolfojcj** [comment]
Hello @snwnst  and @emrebyimrn.

Recently I have been working on a Helm chart to deploy Saleor in a Kubernetes cluster of mine.

At https://artifacthub.io/packages/helm/rc-helm-charts/saleor you will find that chart, which is not official, however may be of help to you.

Regards.
